makefile.shade

```diff
@@ -1,7 +1,34 @@
+use namespace="System.IO"
+
+default BASE_DIR_LOCAL='${Directory.GetCurrentDirectory()}'
+default BUILD_DIR_LOCAL='${Path.Combine(BASE_DIR_LOCAL, "artifacts", "build")}'
+
 var VERSION='0.1'
 var FULL_VERSION='0.1'
 var AUTHORS='Microsoft Open Technologies, Inc.'
 -BuildQuality = "preview3";
 
 use-standard-lifecycle
-k-standard-goals
\ No newline at end of file
+k-standard-goals
+
+#remove-intermediates
+  @{
+    Directory.CreateDirectory(BUILD_DIR_LOCAL);
+    foreach (var p in Files
+      .Include(Path.Combine(BUILD_DIR_LOCAL, "**\\Microsoft.Extensions.*.nupkg"))
+      .Include(Path.Combine(BUILD_DIR_LOCAL, "**\\Microsoft.DotNet.Watcher.Tools.*.nupkg"))
+      )
+    {
+      Log.Info("Deleting " + p);
+      File.Delete(p);
+    }
+  }
+
+#repack-tools .remove-intermediates target='compile' if='Directory.Exists("src") && !IsLinux'
+    @{
+        Dotnet("run -p tools/NuGetPackager -- -c " + E("Configuration") +
```

Comment:
Trying to avoid putting anything in KoreBuild that will be ripped out when we switch to MSBuild....but actually, maybe I should just override the "pack" target completely now that I've [ripped out](https://github.com/aspnet/DotNetTools/pull/177) any "dependency" type packages from this project.